test/Model/EventParticipationTest.php

```diff
@@ -16,8 +14,8 @@ class EventParticipationTest extends \PHPUnit_Framework_TestCase
      */
     public function testSetStatus($status)
     {
-        $event = $this->prophesize(AbstractEvent::class);
-        $user = $this->prophesize(User::class);
+        $event = $this->prophesize("CalendArt\\AbstractEvent");
+        $user = $this->prophesize("CalendArt\\User");
```

Comment:
I wouldn't mind upping the requirements to php 5.6 though :trollface: (but it would need a minor release)